xml/deployment_installation.xml

```diff
@@ -936,20 +936,28 @@ suse_caasp:
       <para>
        Once you have edited the configuration files with your desired settings,
        you need to create an ISO image with the volume label <literal>cidata</literal>
-       containing only the subdirectory for that node type. Use <command>genisoimage</command>
-       to do this.
+       containing only the subdirectory for that node type.
+      </para>
+      <para>
+       On &sle; 12 or &opensuse; 42, use <command>genisoimage</command> to do this.
+       On &sle; or &opensuse; 15, use <command>mkisofs</command>. The parameters
+       are the same for both commands.
       </para>
       <para>
-       To create the ISO image for an admin node:
+       For example, to create the ISO image for an admin node on a computer
+       running &opensuse; 42:
       </para>
       <screen>
 &prompt.user; sudo genisoimage -output <replaceable>cc-admin.img</replaceable> -volid <replaceable>cidata</replaceable> -joliet -rock <replaceable>cc-admin</replaceable>
       </screen>
       <para>
-       To create the ISO image for a worker node, use the same command, but 
-       substitute the name of the folder containing the configuration files for
-       a &worker_node; and title the volume <literal>cc-worker</literal>.
+       To create the ISO image for a worker node on a computer running &opensuse;
+       15, substituting the name of the folder containing the configuration
+       files for a &worker_node; and titling the volume <literal>cc-worker</literal>:
       </para>
+      <screen>
+&prompt.user; sudo mkisofs -output <replaceable>cc-worker.img</replaceable> -volid <replaceable>cidata</replaceable> -joliet -rock <replaceable>cc-worker</replaceable>
```

Comment:
no whitespace after &prompt.user, it's already part of the entity.
tag command as <command>.
wrap the command line with \ if it doesn't fit into a single line in the pdf version.
no line break before the end of the screen